beancount/ingest/importers/csv.py

```diff
@@ -333,6 +331,16 @@ def get(row, ftype):
 
         return entries
 
+    def call_categorizer(self, txn, row):
```

Comment:
Please avoid magic. Just update the interface to require two arguments.
Clients will break; adapting their inputs should be trivial (add an argument, ignore it).

Reply:
I'll merge this now, will remove inspection later.